### PR TITLE
Install generator creates duplicate filename for migrations

### PR DESCRIPTION
**Pull request by PendragonDevelopment**
...on names (line 49 lib/generators/forem/install_generator.rb)

Originally the generator created two files named add_forem_state. Was having an issue installing before I figured out how to fix it.

This is my first pull request, so apologies if I've done something wrong.

### CODE REVIEW

**Comment by PendragonDevelopment** [comment]
Apparently someone else created an issue regarding this (#212). Didn't realize it when I created the pull request.

**Comment by ingus-t** [comment]
/usr/local/rvm/gems/ruby-1.9.3-p125@org/gems/activemodel-3.2.3/lib/active_model/attribute_methods.rb:407:in `method_missing': undefined method`forem_auto_subscribe?' for #User:0x90747d0 (NoMethodError)

This is what I get when trying to install your fork. I have also forked forem and tried to change the faulty line from lib/generators/forem/install_generator.rb to _add_forem_autosubscribe, _add_forem_autosubscribe_to_user, _add_forem_auto_subscribe_to_user . None seem to work for me.

Have you checked if your forked version of forem installs correctly?

**Comment by PendragonDevelopment** [comment]
Got that same error initially. Made this change and it seems to work. Give it a shot?

Edit: Also added Devise just for authorization and model reasons

**Comment by ingus-t** [comment]
This is weird. I understand that the small change you did should fix forem and it should install without a problem. And I have tried specifying
gem ''forem', :git=>"git://github.com/PendragonDevelopment/forem.git"
in my gemfile, running bundle install and then running rails g forem:install.

Installation resulted in error - undefined method forem_auto_subscribe?

**Comment by PendragonDevelopment** [comment]
Yeah, I'm not sure what the issue is. I just ran it again with no errors.

New Rails app ->
Added Devise, Forem, and Will Paginate to gemfile, bundle install
rails g devise:install
rails g Model User name:string (just for the model)
rails g forem:install

Installed with no issues. Is it a new application, or building into an existing one?

**Comment by ingus-t** [comment]
Existing one. Will try again tomorrow. Thanks for the help.

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,223 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
```
Comment (radar) [inline config/initializers/devise.rb]: This file should not exist. Please remove it.

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
```
Comment (radar) [inline config/locales/devise.en.yml]: This file should not exist. Please remove it.

**Comment by PendragonDevelopment** [comment]
Sorry about that, wasn't thinking when I committed that. Did that get it all?

File: Gemfile.lock [hunk 3]
```diff
@@ -47,6 +47,7 @@ GEM
     addressable (2.2.7)
     arel (3.0.2)
     bcrypt-ruby (3.0.1)
+    bcrypt-ruby (3.0.1-x86-mingw32)
```
Comment (radar) [inline Gemfile.lock]: Please remove these modifications to the Gemfile.lock also.

**Comment by PendragonDevelopment** [comment]
It took me a few commits (still not used to the diff view), but I think I got it restored to the original.

**Comment by radar** [comment]
Yup, that's it. Thanks!

**Comment by radar** [comment]
Fixed with @8a90a6f. Thanks again.

**Comment by radar** [comment]
Sorry, that's commit @515839e73bccbcb2f228e23082538bc86e7114fc actually.

**Pull request closed without merging**